pt/refresher-linear-algebra.md

```diff
@@ -1,339 +1,339 @@
 
-&#10230;
+&#10230; Dependência linear - Um conjunto de vetores é dito ser linearmente dependete se um dos vetores no conjunto puder ser definido como uma combinação linear dos demais.
 
 <br>
 
 **40. Remark: if no vector can be written this way, then the vectors are said to be linearly independent**
 
-&#10230;
+&#10230; Observação: se nenhum vetor puder ser escrito dessa maneira, então os vetores são ditos serem linearmente independentes
 
 <br>
 
 **41. Matrix rank ― The rank of a given matrix A is noted rank(A) and is the dimension of the vector space generated by its columns. This is equivalent to the maximum number of linearly independent columns of A.**
 
-&#10230;
+&#10230; Posto da matriz - O posto de uma dada matriz A é indicada por rank(A) e é a dimensão do vetor espaço gerado por suas colunas. Isso é equivalente ao número máximo de colunas linearmente independentes de A.
 
 <br>
 
 **42. Positive semi-definite matrix ― A matrix A∈Rn×n is positive semi-definite (PSD) and is noted A⪰0 if we have:**
 
-&#10230;
+&#10230; Matriz positiva semi-definida - Uma matriz A∈Rn×n é positiva semi-definida (PSD) e é indicada por A⪰0 se tivermos:
 
 <br>
 
 **43. Remark: similarly, a matrix A is said to be positive definite, and is noted A≻0, if it is a PSD matrix which satisfies for all non-zero vector x, xTAx>0.**
 
-&#10230;
+&#10230; Observação: de forma similar, uma matriz A é dita ser positiva definida, e é indicada por A≻0, se ela é uma matriz (PSD) que satisfaz todo vetor x não nulo, xTAx>0.
 
 <br>
 
 **44. Eigenvalue, eigenvector ― Given a matrix A∈Rn×n, λ is said to be an eigenvalue of A if there exists a vector z∈Rn∖{0}, called eigenvector, such that we have:**
 
-&#10230;
+&#10230; Autovalor, autovetor - Dada uma matriz A∈Rn×n, λ é dita ser um autovalor de A se existe um vetor z∈Rn∖{0}, chamado autovetor, nós temos:
 
 <br>
 
 **45. Spectral theorem ― Let A∈Rn×n. If A is symmetric, then A is diagonalizable by a real orthogonal matrix U∈Rn×n. By noting Λ=diag(λ1,...,λn), we have:**
 
-&#10230;
+&#10230; Teorema spectral - Seja A∈Rn×n. Se A é simétrica, então A é diagonalizável por uma matriz ortogonal U∈Rn×n. Indicando Λ=diag(λ1,...,λn), nós temos:
 
 <br>
 
 **46. diagonal**
 
-&#10230;
+&#10230; diagonal
 
 <br>
 
 **47. Singular-value decomposition ― For a given matrix A of dimensions m×n, the singular-value decomposition (SVD) is a factorization technique that guarantees the existence of U m×m unitary, Σ m×n diagonal and V n×n unitary matrices, such that:**
 
-&#10230;
+&#10230; Decomposição em valor singular - Para uma dada matriz A de dimensões mxn, a decomposição em valor singular (SVD) é uma técnica de fatorização que garante a existência de matrizes unitária U m×m, diagonal Σ m×n e unitária V n×n, de tal modo que:
 
 <br>
 
 **48. Matrix calculus**
 
-&#10230;
+&#10230; Cálculo com matriz
```

Comment:
Maybe it would be better `Cálculos` in the plural form??

Reply:
So... this one I don't now... because Calculus in this context I think means `Cálculo`, I mean the study of differential and integral. Then If I use `Cálculos` it would be a more generalized term, don't you think?

Reply:
Agreed. Don't need to change it